proxy/http/HttpProxyServerMain.cc

```diff
@@ -214,7 +215,7 @@ MakeHttpProxyAcceptor(HttpProxyAcceptor &acceptor, HttpProxyPort &port, unsigned
 
     SCOPED_MUTEX_LOCK(lock, ssl_plugin_mutex, this_ethread());
     ssl_plugin_acceptors.push(ssl);
-
+    ssl->port        = &port;
```

Comment:
Probably a bad name, do to punning (e.g. is this a TCP port?). "proxy_port" is a better choice.